demo/tsconfig.node.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "@tsconfig/node22/tsconfig.json",
+  "compilerOptions": {
+    "composite": true,
+    "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.node.tsbuildinfo",
+
+    "baseUrl": ".",
+    "module": "ESNext",
+    "moduleResolution": "Bundler",
+    "types": ["node"],
+    "paths": {
+      "floating-vue/directives": ["../packages/floating-vue/src/directives.ts"],
+      "floating-vue": ["../packages/floating-vue/src/index.ts"]
```

Comment:
we need to add `tsc-alias` as dev dependency, `unbuild` or `tsdown` will resolve these entries, vite requires resolve.alias